x-pack/plugins/ml/server/models/job_audit_messages/job_audit_messages.js

```diff
@@ -39,6 +36,14 @@ const anomalyDetectorTypeFilter = {
   },
 };
 
+function isClearable(index) {
```

Comment:
it deserves some unit tests

Reply:
Added in dfe7484fb36ec61d27085d18e60a46a2807d79df